pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -130,8 +130,17 @@ def _configure_swa_callbacks(self):
             self.trainer.callbacks = [StochasticWeightAveraging()] + self.trainer.callbacks
 
     def configure_progress_bar(self, refresh_rate=None, process_position=0):
-        if os.getenv("COLAB_GPU") and refresh_rate is None:
-            # smaller refresh rate on colab causes crashes, choose a higher value
+        use_rich_progress_bar = _RICH_AVAILABLE
+        if use_rich_progress_bar and refresh_rate is not None:
+            rank_zero_warn(
+                "``RichProgressBar`` does not support setting the refresh rate via the Trainer. "
+                "If you'd like to change the refresh rate and continue using the ``RichProgressBar``, "
+                "please pass ``callbacks=RichProgressBar(refresh_rate=4)``. "
```

Comment:
is it `RichProgressBar(refresh_rate_per_second)` ??
```suggestion
                "please pass ``callbacks=RichProgressBar(refresh_rate_per_second=4)``. "
```

Reply:
I don't think so:
https://github.com/PyTorchLightning/pytorch-lightning/blob/0ffc848dac4ec06839df03ae8d8c4363852651e9/pytorch_lightning/callbacks/progress/rich_progress.py#L138

Reply:
per second can be quite annoying in case you run in Jupyter and it accidentally prints each update on a new line, so you may end up with hundreds of lines without any real change...

Reply:
@Borda this isn't an issue for Rich on colab notebooks, this was only an issue with TQDM